pilot/pkg/kube/inject/webhook_test.go

```diff
@@ -512,6 +565,40 @@ volumes:
 	}
 }
 
+func TestReloadCert(t *testing.T) {
+	wh, cleanup := createWebhook(t, minimalSidecarTemplate)
+	defer cleanup()
+	stop := make(chan struct{})
+	defer func() { close(stop) }()
+	go wh.Run(stop)
+	checkCert(t, wh, testcerts.ServerCert, testcerts.ServerKey)
+	// Update cert/key files.
+	if err := ioutil.WriteFile(wh.certFile, rotatedCert, 0644); err != nil { // nolint: vetshadow
+		cleanup()
+		t.Fatalf("WriteFile(%v) failed: %v", wh.certFile, err)
+	}
+	if err := ioutil.WriteFile(wh.keyFile, rotatedKey, 0644); err != nil { // nolint: vetshadow
+		cleanup()
+		t.Fatalf("WriteFile(%v) failed: %v", wh.keyFile, err)
+	}
+	g := gomega.NewGomegaWithT(t)
+	g.Eventually(func() bool {
+		return checkCert(t, wh, rotatedCert, rotatedKey)
+	}, "10s", "1s").Should(gomega.BeTrue())
+}
+
+func checkCert(t *testing.T, wh *Webhook, cert, key []byte) bool {
```

Comment:
Add a `t.Helper()` here to mark the function as a test helper (see https://golang.org/pkg/testing/#T.Helper). This will improve file/line information in prints.